core/rfb.js

```diff
@@ -69,28 +69,28 @@ export default class RFB extends EventTargetMixin {
 
         // Connection details
         options = options || {};
-        this._rfb_credentials = options.credentials || {};
+        this._rfbCredentials = options.credentials || {};
```

Comment:
Should we see if lint can start enforcing this?

Reply:
Yeah, it can:

https://eslint.org/docs/rules/camelcase#top

I'll create a new PR for enforcing this.

Reply:
#1411